servers/src/main/java/tachyon/master/Master.java

```diff
@@ -20,9 +20,9 @@
 import org.apache.thrift.TProcessor;
 
 import tachyon.master.journal.JournalCheckpointStreamable;
-import tachyon.master.journal.JournalEntry;
 import tachyon.master.journal.JournalInputStream;
 import tachyon.master.journal.ReadWriteJournal;
+import tachyon.proto.JournalEntryProtos.JournalEntry;
```

Comment:
All messages defined by `journal_entry_protos.proto` get placed inside an outer class with the `.proto` name camelCased.